maturity-profiles-policies.md

```diff
@@ -1,26 +1,26 @@
-# .NET Foundation Maturity Ladder Policies
+# .NET Foundation Maturity Profiles Policies

```

Comment:
In the first paragraph this states:

> Any .NET open source project is welcome to participate in the maturity profiles program.

But the heading here says "Policies" I  wonder if we should change this to ".NET Foundation Maturity Profiles Program"